File: prosecurity.controller/src/kz/diploma/prosecurity/controller/model/FileModel.java
package kz.diploma.prosecurity.controller.model;

public class FileModel {
  public String id;
  public String name;
  public String type;
  public String src;

  public FileModel() {
  }

  public FileModel(String id, String name, String type, String src) {
    this.id = id;
    this.name = name;
    this.type = type;
    this.src = src;
  }
}

File: prosecurity.mobile/src/model/file-model.ts
export class FileModel {
  public id: string;
  public type: string;
  public name: string;
  public src: any;


  public static fromFile (f: File): FileModel {
    if (f == null) return null;
    let ret = new FileModel();
    ret.id = null;
    ret.name = f.name;
    //ret.title = f.name;
    ret.type = f.type;
    return ret;
  }

  public static fromFileModel (f: FileModel): FileModel {
    if (f == null) return null;
    let ret = new FileModel();
    ret.id = f.id;
    ret.name = f.name;
    //ret.title = f.title;
    ret.type = f.type;
    ret.src = f.src;

    return ret;
  }

}
